### DISCUSSION TITLE
Backend

### DISCUSSION THREAD

**Discussion by felipe-mendieta** [infra-geo-ouverte/igo2#812] [Q&A] [answered]
Hello coders, I have a question, are there any backend open source for igo2 project? Or any standard for its implementation?
I need connect igo2 to geoserver 
Thanks for the answer.

**Comment by pelord** [comment, marked as answer]
We use geo services from Mapserver, Geoserver, and ESRI with no troubles, handling WMS, WFS, Url based vectors,  Arcgis Feature service, Arcgis Image service and Arcgis Tile Service, MVT, WMTS, XYZ ....  The are other connectors available (CartoDB, websocket) but I dont use them.. 

Hope it's help?

**Comment by pelord** [comment]
Now, strictly talking about backend, our geo services are connected to various database... we use postgresql.
